generator/raw_out.py

```diff
@@ -0,0 +1,33 @@
+import re
+from collections import defaultdict
+from typing import DefaultDict
+from typing import List
+
+import requests
+
+TERM_URL = 'https://raw.githubusercontent.com/togakangaroo/tech-terms/master/terms.org'
+
+
+def convert_to_dict_list(items: List[dict]) -> DefaultDict[str, list]:
```

Comment:
so for my own understanding, any reason not to do this as a dictionary generator?

      return {x['term']: x['definition'] for x in items}

You would then not need to pass in a list either, you could keep it a sequence and wouldn't have to double-load into memory.

Reply:
Sounds good.